scripts/bosh-cli.sh

```diff
@@ -2,8 +2,12 @@
 
 set -eu
 
-BOSH_ID_RSA="$(aws s3 cp "s3://gds-paas-${DEPLOY_ENV}-state/id_rsa" - | base64)"
-export BOSH_ID_RSA
+if [ -z "${SSH_PATH}" ]; then
```

Comment:
Could make this a one-liner like we do elsewhere in the code base: `SSH_PATH=${SSH_PATH:-"/Users/${USER}/.ssh/id_rsa"}` or something to that effect.